src/main/resources/reference.conf

```diff
@@ -134,6 +152,24 @@ cassandra-snapshot-store {
   # Delay between connection retries
   connect-retry-delay = 5s
 
+  # Cassandra connection connections per host core local
+  connections-per-host-core-local = 2
+
+  # Cassandra connection connections per host max local
+  connections-per-host-max-local = 8
```

Comment:
is this because snapshots can be large and it would be beneficial to use several connections for that?
when is a new connection established? is that when 32768 is exceeded?